include/vinecopulib/vinecop/implementation/class.ipp

```diff
@@ -727,17 +730,9 @@ Vinecop::cdf(const Eigen::MatrixXd &u, const size_t N,
     check_data_dim(u);
 
     // Simulate N quasi-random numbers from the vine model
-    Eigen::MatrixXd U(N, d_);
-    if (d_ > 300) {
-        U = tools_stats::sobol(N, d_, seeds);
-    } else {
-        U = tools_stats::ghalton(N, d_, seeds);
-    }
+    auto U = tools_stats::simulate_uniform(N, d_, true, seeds);
```

Comment:
Maybe here it should be switched from U to u too no?

Reply:
Or something else since u is already taken.

Reply:
u_sim?

Reply:
Sure